fsleyes_plugin_shimming_toolbox/utils.py

```diff
@@ -1,24 +1,30 @@
 import subprocess
 import logging
+
+from pathlib import Path
+
 logger = logging.getLogger(__name__)
 
+HOME_DIR = str(Path.home())
+PATH_ST_VENV = f"{HOME_DIR}/shimming-toolbox/python/envs/st_venv/bin"
+
 
-# TODO: find a better way to include this as it is defined in utils as well
 def run_subprocess(cmd):
     """Wrapper for ``subprocess.run()`` that enables to input ``cmd`` as a full string
         (easier for debugging).
 
     Args:
         cmd (string): full command to be run on the command line
     """
-    logging.debug(f'{cmd}')
+    logging.info(f'{cmd}')
     try:
         subprocess.run(
             cmd.split(' '),
             stdout=subprocess.PIPE,
             stderr=subprocess.PIPE,
             text=True,
-            check=True
+            check=True,
+            env=dict(PATH=PATH_ST_VENV)
```

Comment:
I think it would be alright! But I would like to better understand the root cause, too, since it may reveal other issues with the ST installation method. :thinking:

Reply:
> The weirdest thing is that the shebang in the `st_b1shim` launcher (and all the other launchers I guess) is only ignored when it links to `st_venv`

Aha! Thank you for reminding me of this. I think I have an alternate solution:

```suggestion
            shell=True
```

From: https://stackoverflow.com/a/6441590

>

Reply:
That's one of the methods I tried before raising the issue and the shebang was still ignored... Moreover, I saw [here ](https://stackoverflow.com/a/3172690/14061141) that the use of `shell=True`  should be done cautiously. I already spent a lot of time dealing with this problem and the best solution I came up with was to use `env=dict(PATH=PATH_ST_VENV)`, but just like you, I'd like to understand why the other fixes do not work...

Reply:
Ah, my apologies. I wasn't aware of what you have and haven't tried.